Dockerfile.frontend

```
@@ -0,0 +1,21 @@
+# Uses a multi-stage container build to build the frontend.
+# Base and builder image will need to be replaced by Fips compliant one
+FROM registry.access.redhat.com/ubi8/ubi:8.9-1136 AS builder
+RUN yum update -y && yum install -y tar make && yum clean all
+RUN curl -L https://go.dev/dl/go1.22.1.linux-amd64.tar.gz -o go.tar.gz \
+    && tar -C /usr/local -xzf go.tar.gz \
+    && rm go.tar.gz
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV GOPATH="/go"
+ENV PATH="${GOPATH}/bin:${PATH}"
+
+WORKDIR /app
+
+COPY . /app
+RUN ls -a ./ && make frontend
```

```suggestion
RUN make frontend
```
Is there a reason to list out the files first?

---

ah i forgot to delete the ls i put to debug the build